app/components/ui/CalculatorInput.tsx

```diff
@@ -162,21 +162,21 @@ const CalculatorInput = () => {
                       onValueChange={(value) => field.onChange(value)} // Bind selection to field
                     >
                       <div className="flex items-center space-x-2">
-                        <RadioGroupItem value="D" id="option-one" />
-                        <Label htmlFor="option-one">Detached</Label>
+                        <RadioGroupItem value="D" id="radio_option_D" />
```

Comment:
```suggestion
                        <RadioGroupItem value="D" id="radio-option-D" />
```

We should stick to kebab-case for these variables 👍

Reply:
Thanks for the quick review!